lms/djangoapps/verify_student/views.py

```diff
@@ -1312,8 +1312,8 @@ def results_callback(request):
     checkpoints = VerificationCheckpoint.objects.filter(photo_verification=attempt).all()
     VerificationStatus.add_status_from_checkpoints(checkpoints=checkpoints, user=attempt.user, status=status)
 
-    # If this is re-verification then send the update email
-    if checkpoints:
+    # Trigger ICRV email only if feature flag for ICRV emails is set
+    if settings.FEATURES.get('ENABLE_ICRV_STATUS_EMAILS', False) and checkpoints:
```

Comment:
This change is untested. Please add a test verifying that the email is not sent when the feature flag is disabled.